bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -110,6 +110,9 @@ spec:
           {{- if .Values.postgresql.args }}
           args: {{- include "common.tplvalues.render" (dict "value" .Values.postgresql.args "context" $) | nindent 12 }}
           {{- end }}
+          {{- if .Values.metrics.customMetrics }}
+          args: ["--extend.query-path", "/conf/custom-metrics.yaml"]
```

Comment:
I thought the same thing. If the user provides a custom image, there is no guarantee that metrics will work, so the custom args have to be set anyway according to the image used.

What would fit principle of the lease surprise better:
- Having the args vector joined if customMetrics and args are set?
- Commenting in the Readme on args vector that customisation will lead to full replacement of args?

I think since normal behaviour of helm is not merging vectors but full replacement of fields second way makes more sense?

Reply:
Line 113 is the wrong place since its repmnr and not exporter configuration. So the args are not conflicting. Ignore my previous comment. Changed the PR according to that.